### PR TITLE
The Farmening Part 12 ("Technobug should be making this PR but he threw a temper tantrum because I said something about his thighs and now I'm doing it instead and it's probably going to be better than if he did it so it was ultimately a net positive" edition)

### PR DESCRIPTION
**Pull request by Hekzder**
## About The Pull Request

"Fixes" the fact that our plants take like three minutes to grow, drastically increases the time it takes for them to grow across the board but also decreases nutrient drain quite dramatically to achieve what I think Techno was originally intending in that farming takes a while but is generally passive and you don't have to babysit. Also fixes a few farming related bugs with icons and the fact that all plants would start taking damage after a while because there was a faulty light check. 

Adds fertilizer and the ability to craft it, as well as a "farming item spawner" with farming relevant items placed in farming related locations on the map and also added to the general hardware spawner with a 5% chance to replace the fishing rod spawner. I included a lot of annotations so hopefully the understanding of agriculture code moving forward can be better for everyone. 

Basically TLDR is: 

- Farming takes longer but is more passive
- Output increased slightly to compensate
- Plants that can be harvested multiple times are balanced around such, they take a very long time to get the full use out of them but they will give you more than single harvest plants overall. 
- Farming item spawner with working fertilizer. 

## Why It's Good For The Game

Farming is whack as freak right now and this fixes it so our other features like molotovs and smigglerettes are also more balanced.

### CODE REVIEW

**Comment by Hekzder** [comment]
This is now also the Flaskening. Adds a lot more herbal remedies to complement improved farming. Descriptions incoming tomorrow

**Comment by TheRealScarHomie** [comment]
dr faucis dark datura flask

**Comment by ChungusGamer666** [comment]
oh my fauci, non-GMO organically grown crops!

**Comment by Hekzder** [comment]
Harvest sack added, fwoah!!

**Comment by Hekzder** [comment]
This now also adds our new griddles and some cool sounds for them. Along with some slight fixes that I saw needed to be done while poking my head around and remembering stuff. I really don't want to add anything else to this to make it more unatomized so please God merge me
